3scale/3scale-operator
# THREESCALE-7961: Use sum_irate instead of sum_rate

## Description

> Link to JIRA: https://issues.redhat.com/browse/THREESCALE-7961

Add logic to use `sum_irate` in monitoring queries when the Openshift version is greater or equal than 4.9. Keep using `sum_rate` when the Openshift version is less than 4.9.

## Verification steps

1. **On an Openshift 4.9 (or above) cluster**
1. Install 3scale operator from this PR and the monitoring stack
2. Create an APIManager with monitoring enabled
3. Verify that the Grafana dashboards that use `sum_rate` use now `sum_irate` and show data
4. **On an Openshift 4.8 (or below) cluster**
5. Repeat steps 2 and 3
6. Verify that the Grafana dashboards continue using `sum_rate` and show data
